src/main/scala/com/codacy/api/client/CodacyClient.scala

```diff
@@ -97,6 +97,7 @@ class CodacyClient(
     } catch {
       case NonFatal(ex) =>
         if (numRetries.exists(x => x > 0)) {
+          sleepTime.map(x => Thread.sleep(x))
           post(request, value, timeoutOpt, sleepTime, numRetries.map(x => x - 1))
```

Comment:
but do we want to sleep here? What does the sleepTime that we already propagate to the post do? Do you have any input @lolgab ?